readme.md

```diff
@@ -465,11 +465,11 @@ Invoices
 ### Create a customer invoice
 
 	var invoiceService = new StripeInvoiceService();
-	StripeInvoice response = invoiceService.Create(*customerId*);
+	StripeInvoice response = invoiceService.Create(*customerId*, stripeInvoiceCreateOptions);
```

Comment:
modified this to be like the other ones (convention):
StripeInvoice response = invoiceService.Create(*customerId*); // optional StripeInvoiceCreateOptions